pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -63,3 +81,6 @@ def create_pep_zero(_: Sphinx, env: BuildEnvironment, docnames: list[str]) -> No
     docnames.insert(1, pep_zero_filename)
     # Add to files for writer
     env.found_docs.add(pep_zero_filename)
+
+    # Create peps.json
+    create_pep_json(peps, app.outdir)
```

Comment:
Might be cleaner to do all I/O in this function and have `create_pep_json` just return a string?

Reply:
Yeah, I think maybe so since we're hardcoding the output file name. But I leave it to @hugovk 's judgement.